sessions/recap-project-6/example-implementation-with-comments/components/StyledLink.js

```diff
@@ -0,0 +1,25 @@
+import styled, { css } from "styled-components";
+
+export const StyledLink = styled.a`
```

Comment:
we can use Link here instead
```suggestion
import Link from 'next/link'

export const StyledLink = styled(Link)`
```